packages/token/src/tests/erc1155.cairo

```diff
@@ -1,10 +1,4 @@
-pub mod common;
-
-#[cfg(test)]
 mod test_dual1155;
-#[cfg(test)]
 mod test_dual1155_receiver;
-#[cfg(test)]
 mod test_erc1155;
-#[cfg(test)]
```

Comment:
I assume the `cfg` is redundant because they're declared as such in token/src/tests?